sdk/synapse/synapse-access-control/package.json

```diff
@@ -28,17 +28,49 @@
   "module": "./dist-esm/src/index.js",
   "types": "./types/synapse-access-control.d.ts",
   "devDependencies": {
-    "typescript": "~4.2.0",
-    "eslint": "^7.15.0",
-    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/identity": "2.0.0-beta.5",
+    "@azure/test-utils-recorder": "^1.0.0",
+    "@microsoft/api-extractor": "7.7.11",
+    "@rollup/plugin-commonjs": "11.0.2",
+    "@types/chai": "^4.1.6",
+    "@types/mocha": "^7.0.2",
+    "chai": "^4.2.0",
+    "dotenv": "^8.2.0",
+    "eslint": "^7.15.0",
+    "karma-chrome-launcher": "^3.0.0",
+    "karma-coverage": "^2.0.0",
+    "karma-env-preprocessor": "^0.1.1",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-ie-launcher": "^1.0.0",
+    "karma-junit-reporter": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-mocha": "^2.0.1",
+    "karma-sourcemap-loader": "^0.3.8",
+    "karma": "^6.2.0",
+    "mocha-junit-reporter": "^1.18.0",
+    "mocha": "^7.1.1",
+    "nyc": "^14.0.0",
+    "prettier": "^1.16.4",
     "rimraf": "^3.0.0",
-    "rollup": "^1.16.3",
     "rollup-plugin-node-resolve": "^3.4.0",
```

Comment:
is this expected to be overwritten in the future by code gen? if not, and since the dev-tool's shared rollup config is being used, all rollup plugin deps here could be removed, and only rollup itself is needed as a dep.